test/org/jetbrains/plugins/ideavim/extension/exchange/VimExchangeExtensionTest.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * IdeaVim - Vim emulator for IDEs based on the IntelliJ platform
+ * Copyright (C) 2003-2020 The IdeaVim authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.jetbrains.plugins.ideavim.extension.exchange
+
+import com.maddyhome.idea.vim.command.CommandState
+import com.maddyhome.idea.vim.helper.StringHelper
+import com.maddyhome.idea.vim.helper.VimBehaviorDiffers
+import org.jetbrains.plugins.ideavim.VimTestCase
+
+class VimExchangeExtensionTest: VimTestCase() {
+  @Throws(Exception::class)
+  override fun setUp() {
+    super.setUp()
+    enableExtensions("exchange")
+  }
+  // |cx|
+  fun `test exchange words left to right`() {
+    doTest(StringHelper.parseKeys("cxe", "w", "cxe"),
+      "The quick <caret>brown fox catch over the lazy dog",
```

Comment:
JFI, `VimTestCase` has some constants like `c` that can be used instead of `<caret> in the tests. E.g. this like can look like `"The quick ${c}brown fox catch over the lazy dog"`. It doesn't provide any features except that it's now highlighted in IJ and thus easy to recognize.